mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -7,11 +7,15 @@
 import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 import static org.springframework.security.jwt.JwtHelper.decodeAndVerify;
 
+import java.io.ByteArrayOutputStream;
 import java.io.FileReader;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.security.MessageDigest;
```

Comment:
a lot of these new imports are now unnecessary, right?

Reply:
yup cleaning.